float32 scale
int width
int height

int8[] occupancyGrid
bool[] deltaOccupancyGrid